app/models/comment.rb

```diff
@@ -1,2 +1,16 @@
 class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  belongs_to :post
+
+  validates :author_id, presence: true
+  validates :post_id, presence: true
+  validates :text, presence: true
+
+  after_save :update_comments
+
+  def update_comments
+    post = self.post
+    post.update(comments_counter: post.comments.count)
+    comments.save
+  end
```

Comment:
- [ ] @GhostEsso Great work implementing the change requested, however, to make it work correctly, kindly consider making the `update_comments` method private.